docs/semgrep-ci/network-broker.md

```diff
@@ -45,32 +45,32 @@ Ensure that you are logged in to the server where you want to run Semgrep Networ
       url: http://[fdf0:59dc:33cf:9be9:0000:0000:0000:0001]/ping
     allowlist: []
     gitlab:
-      baseUrl: <https://gitlab.xxxx.net/api/v4>
+      baseUrl: YOUR_BASE_URL
       token: YOUR_TOKEN
   ```
 
-  The `publicKey` should be entered precisely as follows:
+  The `publicKey` value should be entered precisely as follows:
 
   ```console
   4EqJwDZ8X/qXB5u3Wpo2cxnKlysec93uhRvGWPix0lg=
   ```
 
-1. The broker requires a WireGuard keypair to establish a secure connection. To generate the public key:
+1. The broker requires a WireGuard keypair to establish a secure connection. To generate your private key `YOUR_PRIVATE_KEY`:
 
    1. Determine the [network broker version](https://github.com/semgrep/semgrep-network-broker/pkgs/container/semgrep-network-broker) you want to use. The format should be similar to `v0.14.0`.
 
-   1. Run the following command in the CLI to generate the public key, replacing the placeholder with the network broker version number:
+   1. Run the following command in the CLI to generate your private key, replacing the placeholder with the network broker version number:
   <pre class="language-console"><code>docker run ghcr.io/semgrep/semgrep-network-broker:<span className="placeholder">VERSION_NUMBER</span> genkey</code></pre>
 
-1. Run the following command in the CLI to generate the private key, replacing the placeholders with the public key you generated in the previous step and the network broker version number:
+1. Run the following command in the CLI to generate your public key, replacing the placeholders with your private key generated in the previous step and the network broker version number:
 
-  <pre class="language-console"><code>echo `<span className="placeholder">PUBLIC_KEY</span>` | sudo docker run -i ghcr.io/semgrep/semgrep-network-broker:<span className="placeholder">VERSION_NUMBER</span> pubkey</code></pre>
+  <pre class="language-console"><code>echo `<span className="placeholder">YOUR_PRIVATE_KEY</span>` | sudo docker run -i ghcr.io/semgrep/semgrep-network-broker:<span className="placeholder">VERSION_NUMBER</span> pubkey</code></pre>
 
   :::info Key sharing 
   Your public key is safe to share. Do **not** share your private key with anyone, including Semgrep.
   :::
 
-1. Update the `config.yaml` file with the private key:
+1. Update the `config.yaml` file with your private key generated in step 2:
```

Comment:
```suggestion
1. Update the `config.yaml` file with your private key generated in a previous step:
```

I think we should make this vague in case we update steps in the future

Reply:
Updated by removing reference to previous step